ocrd_tool.md

```diff
@@ -59,6 +73,16 @@ properties:
           executable:
             description: The name of the CLI executable in $PATH
             type: string
+          input_file_grp:
+            description: Input fileGrp@USE this tool expects by default
+            type: array
+            items:
+              type: string
+              pattern: '^OCR-D-[A-Z0-9-]+$'
```

Comment:
Typo?: [A-Z0-9\\-]

Reply:
dash at the end of a character group is not interpreted as a meta char, c.f. https://regex101.com/r/4d8P1b/1.

Reply:
Ok, I didn't know that yet.